core/server/services/bulk-email/index.js

```diff
@@ -46,24 +47,38 @@ module.exports = {
      * @param {Email} message - The message to send
      * @param {[EmailAddress]} recipients - the recipients to send the email to
      * @param {[object]} recipientData - list of data keyed by email to inject into the email
-     * @returns {Promise<boolean>} A promise representing the success of the email sending
+     * @returns {Array{Promise<boolean>}} An array of promises representing the success of the batch email sending
```

Comment:
Minor, but i'm pretty sure this should be `Promise<Array<object>>` now

Reply:
Yeap. My memory failed me here :)